### PR TITLE
WD-2337 Update /landscape page

### PR DESCRIPTION
**Pull request by carkod**
## Done

Copy update

## QA

- Check https://ubuntu-com-12622.demos.haus/landscape against [copy doc](https://docs.google.com/document/d/1Q5KPVOn3yaexgSbkZHmXHmWzT5WG3ocYAPthweMwDz4/edit#heading=h.fmc57n38rjz9)

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-2337?atlOrigin=eyJpIjoiNDNmNzY4YzgyZDQxNDcwMDhmY2MzZGYwOWIzN2Q5NTUiLCJwIjoiaiJ9

## Help

[QA steps](https://discourse.canonical.com/t/qa-steps/152) - [Commit guidelines](https://discourse.canonical.com/t/commit-guidelines/148)

### CODE REVIEW

File: templates/landscape/index.html [hunk 1]
```diff
@@ -3,6 +3,7 @@
 {% block title %}Landscape{% endblock %}
 {% block meta_description %}Landscape is the leading management and administration tool for all versions of Ubuntu, anywhere.{% endblock meta_description %}
 {% block meta_copydoc %}https://docs.google.com/document/d/1Q5KPVOn3yaexgSbkZHmXHmWzT5WG3ocYAPthweMwDz4/edit{% endblock meta_copydoc %}
+{% block meta_image %}https://assets.ubuntu.com/v1/742fa591-Canonical-Landscape-logo-2022.png{% endblock meta_image %}
```
Comment (tbille) [inline templates/landscape/index.html:6]: This format of meta image is not supported
You can test here: https://www.heymeta.com/url/ubuntu-com-12622.demos.haus/landscape

image

Comment (carkod) [reply templates/landscape/index.html:6]: What do you mean the format of the image? .png is not supported? or is it the size of the image?

We may need to ask a designer then to create one custom for this.

Comment (carkod) [reply templates/landscape/index.html:6]: Checking here I don't see any issues, but the image size does get cropped https://www.opengraph.xyz/url/https%3A%2F%2Fubuntu-com-12622.demos.haus%2Flandscape

Comment (tbille) [reply templates/landscape/index.html:6]: @carkod just got an asset for you! https://assets.ubuntu.com/v1/6b9a0b06-landscape-metatag.svg

Comment (carkod) [reply templates/landscape/index.html]: Updated. Thanks.

**Review by tbille — APPROVED** [review]

**Pull request merged**